### PR TITLE
Update message about Default Execution Environment

### PR DESCRIPTION
**Pull request by nixocio**
Update message about Default Execution Environment

Organizations

image

Projects

image

See: https://github.com/ansible/awx/issues/9715

### CODE REVIEW

**Comment by shanemcd** [comment]
Should we say something like "jobs that _use_ this project"?

**Comment by shanemcd** [comment]
Maybe something like this:

> The Execution Environment that will be used for jobs that use this project. This will be used a fallback when an EE has not been explicitly assigned at the Job Template or Workflow level.

Will defer to @wenottingham and @tvo318 about the capitalization of resource types here.

**Comment by jbradberry** [comment]
I think, barring feedback from @wenottingham and @tvo318 otherwise, that @shanemcd 's last suggestion (https://github.com/ansible/awx/pull/10278#issuecomment-848783949) should be used.

**Comment by nixocio** [comment]
@jbradberry @shanemcd updated as per suggestions.

**Comment by nixocio** [comment]
regate

File: awx/ui_next/src/screens/Organization/shared/OrganizationForm.jsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ function OrganizationFormFields({
         onBlur={() => executionEnvironmentHelpers.setTouched()}
         value={executionEnvironmentField.value}
         onChange={value => executionEnvironmentHelpers.setValue(value)}
-        popoverContent={t`Select the default execution environment for this organization.`}
+        popoverContent={t`The Execution Environment that will be used for jobs inside of this organization. This will be used a fallback when an EE has not been explicitly assigned at the Project, Job Template or Workflow level.`}
```
Comment (wenottingham) [inline awx/ui_next/src/screens/Organization/shared/OrganizationForm.jsx]: In general, Execution Environment is not capitalized... but neither would be Project, Job Template, or Workflow. So... meh.

I would not use 'EE' - would spell it out.

Comment (nixocio) [reply awx/ui_next/src/screens/Organization/shared/OrganizationForm.jsx]: @wenottingham updated.

**Review by wenottingham — APPROVED** [review]

**Pull request merged**